.travis.yml

```diff
@@ -6,7 +6,6 @@ scala:
 sudo: required
 jdk:
   - oraclejdk8
-dist: trusty
```

Comment:
What's the purpose of this change?

Reply:
It was generated by Intelij. Will remove